airbyte-integrations/connectors/source-outbrain-amplify/source_outbrain_amplify/schemas/budgets_for_marketers.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
```

Comment:
Can you add addtionalProperties and set it to true for each of the schemas? This will help make the connector less prone to breaking with future api changes.